src/ripple/protocol/Indexes.h

```diff
@@ -263,6 +263,10 @@ nft_buys(uint256 const& id) noexcept;
 Keylet
 nft_sells(uint256 const& id) noexcept;
 
+/** AMM entry */
+Keylet
+amm(uint256 const& amm) noexcept;
```

Comment:
I'd expect amm keylets to be computed from two issues. Why is it computed like this instead?

Reply:
Changed.